table/pit-cs_test.go

```diff
@@ -0,0 +1,156 @@
+package table
```

Comment:
Thank you for write these tests. But to be honest they are not quite useful: it is almost impossible for a simple getter to fail.
Tests in `pit-cs-tree_test` are more important: we want to confirm the insertion and matching are correct.

Reply:
The getter/setter tests were the first tests I wrote, mainly to get used to writing unit tests in Go and increase test coverage. I'll finish the unit tests in `pit-cs-tree_test.go` this week.

Reply:
I see. Thanks.